contracts/lease/src/api/open/mod.rs

```diff
@@ -28,6 +31,9 @@ pub struct NewLeaseForm {
     pub customer: Addr,
     /// Ticker of the currency this lease will be about.
     pub currency: SymbolOwned,
+    /// Optional parameter indicating preferred maximum amount of the currency
+    /// the loan should be opened with.
+    pub max_loan: Option<CoinAmount>,
```

Comment:
After a thorough analysis of the requirements and the "ease-of-use" and "intuitive" interface principles, let's change the input to `max_ltv: Option<Percent>` with doc string:
`Maximum Loan-to-Value percentage of the new lease, optional`